### PR TITLE
Fix monkey yaml's handling of carriage return

### LINKED ISSUE #295: Can't find propertyHelper.js
I just ran test262 with the SpiderMonkey shell, however during the run I get the following error:
`Error: Can't find: /home/tom/projects/test262/harness/propertyHelper.js`
However manually looking up this path the file exists.

### PR DESCRIPTION
**Pull request by arv**
monkeyYaml didn't split lines correctly leading to \r in resulting
values.

Fixes #295

### CODE REVIEW

**Comment by arv** [comment]
@bterlson @anba @smikes PTAL

File: tools/packaging/test/test_monkeyYaml.py [hunk 1]
```diff
@@ -106,6 +106,10 @@ def test_multiline_list3(self):
         self.assertEqual(lines, ["baz: bletch"])
         self.assertEqual(value, ["foo", "bar"])
 
+    def test_mutliline_list_carriage_return(self):
```
Comment (anba) [inline tools/packaging/test/test_monkeyYaml.py]: nit: s/mutliline/multiline/

Comment (arv) [inline tools/packaging/test/test_monkeyYaml.py]: Done... amended.

**Comment by arv** [comment]
Ping. Can someone with permission please take a look? This is preventing us from pulling down a newer version of test262

**Comment by bterlson** [comment]
Thanks for the ping, sorry for the delay. LGTM!

**Comment by arv** [comment]
Thanks

**Pull request merged**